test/new-e2e/tests/agent-metrics-logs/log-agent/linux-log/journald/log-config/include.yaml

```diff
@@ -4,4 +4,4 @@ logs:
     source: custom_log
     service: random-logger
     include_units:
-      random-logger.service
\ No newline at end of file
+      random-logger.service
```

Comment:
Would it be possible to avoid non-functional changes?

Reply:
hm I'm surprised by this change, this was not intentional I'll fix this